packages/block-library/src/social-link/editor.scss

```diff
@@ -1,6 +1,7 @@
 .wp-block-social-links .wp-social-link button {
 	color: currentColor;
 	padding: 6px;
+	height: var(--wp-social-link-size);
```

Comment:
Is it appropriate to surfaces this as a CSS variable even if it's not part of global styles? @nosolosw 

I think I'd prefer we avoid CSS variables entirely unless it's directly tied to global styles. An alternative is to have CSS classes, `is-small`, `is-large`, `is-huge` instead, and then apply the sizing based on that.

Reply:
My expectation was that at some point themes might be able to modify this via global sizes — similar to how they can provide font size presets. 

That may be overkill from a design tools perspective, but I figured it can't hurt to allow that.

Reply:
Theme-abillity through global styles is definitely a nice feature, and if that requires these to be CSS variables (and if this is the correct way to do that), then I'm all thumbs up from here.

Reply:
I'd probably go with raw values instead of CSS variables in this PR. There's nothing wrong with using CSS vars as they are here (it's actually a nice use case). My only concern is that people start using them to tweak social icons so they become public API and we can't modify how this works down the road when/if this is integrated in Global Styles proper.

For some context: so far, the styles that can be declared via Global Styles are a straightforward 1:1 mapping to some CSS property (eg: fontSize, backgroundColor, etc). The needs here are a bit different in that four things need to be coordinated at once (width/height of the link, width/height of the logo), which makes that behaviour more particular to this specific block and less as a general utility (unlike fontSize or backgroundColor). This is not to say it can't be done, but that I see a few challenges that merit its own separate discussion.

Reply:
Yes, the idea using CSS variables was that a theme would be able to provide a custom value or preset.